weblate/static/style-bootstrap.css

```diff
@@ -1101,10 +1107,6 @@ thead th {
   border: 1px solid #2eccaa;
 }
 
-.btn-link {
```

Comment:
This class is used in many other locations, I'd expect this breaks rendering there.

Reply:
Indeed. it is used on many elements. Also in the css there are multiple definitions of this class. I assume the default padding applied by bootstrap is enough. But I will leave the class styles here and remove it from the element instead.

Reply:
Removing the CSS makes these padding smaller, look at zen mode.

Before this PR:

![obrazek](https://github.com/user-attachments/assets/3e865960-815f-4768-b45c-4fe01504d964)

With this PR applied:

![obrazek](https://github.com/user-attachments/assets/79a960d3-1376-4668-a30e-e54fc97eb9ef)